src/utils/flteredPeopleList.ts

```diff
@@ -0,0 +1,46 @@
+import { Person } from '../types';
+
+export const filteredPeople: React.FC<Person> = (currentPeople, search) => {
```

Comment:
```suggestion
export const getFilteredPeople: React.FC<Person> = (currentPeople, search) => {
```
`filteredPeople` sounds like it is already the array of filtered people
use verbs in function names

Reply:
Not resolved